microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md

```diff
@@ -45,6 +48,36 @@ The top of each context menu now shows the title again:
 ![New context menu header](./devtools-128-images/header-title.png)
 
 
+<!-- ====================================================================== -->
+## Heap snapshot improvements
+
+<!-- Subtitle: Heap snapshots are generated faster, HTML elements are grouped by tag name, shallow sizes show fewer internal objects, and numeric values are always shown. -->
+
+In the **Memory** tool, there are several improvements for heap snapshots:
+
+* Generating heap snapshots is now faster.
+
+* HTML elements are grouped by tag name.<!-- todo: by element name? -->
```

Comment:
```suggestion
* HTML elements are grouped by element name.
```
alts:
* by element name
* by the starting tab of the element
* by the element name
* by the tag name

somehow i find the existing wording unclear, need to see UI to clarify wording

Reply:
I meant HTML tags. ex: <img>, <div>, <body>, etc.

Reply:
result:
```
* HTML elements are grouped by tag name, 
such as `<body>`, `<div>`, or `<img>`.
```